### DISCUSSION TITLE
Project status_

### DISCUSSION THREAD

**Discussion by pewterbrass** [unicorn-engine/unicorn#1997] [General]
Is Unicorn still actively being developed? If so, by who?

**Comment by wtdcode** [comment]
Yes, I'm maintaining Unicorn at this moment.

**Comment by VelocityRa** [comment]
> Yes, I'm maintaining Unicorn at this moment.

Is that still the case? Last commit was 4 months ago.

**Reply by wtdcode** [reply]
https://github.com/unicorn-engine/unicorn/issues/2071

**Reply by wtdcode** [reply]
And you should check dev branch
